### PR TITLE
Less -> Scss migration

### PR DESCRIPTION
**Pull request by babich-a**

### CODE REVIEW

File: scss/bundles/dx.carmine.compact.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@use "../widgets/generic" with (
+$color: "carmine",
+$size: "compact"
```
Comment (San4es) [inline scss/bundles/dx.carmine.compact.scss:3]: Should there be an indent here?

File: scss/widgets/base/_colorView.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+@mixin dx-base-colorview-styles($handle-color, $handle-border-color) {
+    .dx-colorview-palette-handle {
```
Comment (San4es) [inline scss/widgets/base/_colorView.scss]: I think we should use 2 spaces like in TS.

Comment (babich-a) [reply scss/widgets/base/_colorView.scss]: https://github.com/DevExpress/DevExtreme/pull/13518/commits/d1edef37cf34f88f81767446af07f74e38914135

File: scss/bundles/dx.carmine.compact.scss (hunk 1 above)
Comment (babich-a) [reply scss/bundles/dx.carmine.compact.scss:3]: I have plans to manually refactor all scss after generator removing.

**Review by vconst — COMMENTED** [review]
Why variables in ssas have different styles?
$filterbuilder-plus-icon-color
$hoverColor, $focusColor
$icon_size, $container_size
$RECURRENCE_RADIO_GROUP_ITEM_PADDING
4 types of naming.

**Comment by babich-a** [comment]
> 
> 
> Why variables in ssas have different styles?
> $filterbuilder-plus-icon-color
> $hoverColor, $focusColor
> $icon_size, $container_size
> $RECURRENCE_RADIO_GROUP_ITEM_PADDING
> 4 types of naming.

Why less have 4 types of naming? We have plans to refactor it. Now it is the result of auto-generation. All variable names are the same.

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**